bigbone/src/main/kotlin/social/bigbone/MastodonClient.kt

```diff
@@ -1014,25 +1014,15 @@ class MastodonClient private constructor(
             )
         }
 
-        @Throws(InstanceVersionRetrievalException::class)
```

Comment:
Instead of removing it, I will use this one where we try to get the version in `MastodonClient`. 

`NodeInfoClient` deals with the server information itself so I think there, the existing exceptions make sense. But I see value in returning the more specific `InstanceVersionRetrievalException` in `MastodonClient` if we cannot find the `version` within the server info returned.

Reply:
Sounds good to me 👍